docs/tutorial/03-model-serving.ipynb

```diff
@@ -26,7 +26,7 @@
     "- [**Build a custom serving class**](#custom-class)\n",
     "- [**Building advanced model serving graph**](#serving=graph)\n",
     "\n",
-    "{octicon}`video` [**Watch the video tutorial**](https://youtu.be/OUjOus4dZfw).\n",
+    "[**Watch the video tutorial**](https://youtu.be/OUjOus4dZfw).\n",
```

Comment:
Was there a reason to remove the graphic? Pls return it unless there's some problem.